package.json

```diff
@@ -12,7 +12,8 @@
     "@tauri-apps/api": "^1.0.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-snap": "^1.23.0"
+    "react-snap": "^1.23.0",
```

Comment:
@satrams my apologies. I think I introduced this into the code base while testing something different